### ISSUE TITLE
Enhancement:  Bring the old charter back.

### ISSUE THREAD

**Issue by zero-cache** [KadeArchive/Kade-Engine#1493]
Please add a way to bring the old charter back the new one is absolutely broken.
It loads for extremely long and the note placement is crappy, I cannot save, I still hear voice files after disabling them in the editor.

I wish we can get the old one back.

**Comment by Spel0** [comment]
>it loads for extremely long

True, kade forgot to add grid removal feature so it would load faster

>note placement is buggy

Press shift + left or right to change snap value. Less snap = less diversity from grid you can do

>can't save the chart

What do you mean? The save button is right near song name in third section

>I still hear voice track after disabling it

Voice track always should play tho? Only thing you can disable is instrumental track. Or do you mean after pressing space to pause? Never encountered that when charting

**Comment by Burger-Gaming** [comment]
the chart can be saved but it frequently glitches when you select it on freeplay, presumably from diffuculty calculation. the song out right crashes if you save it and then open it through the chart menu

**Comment by Rdac0** [comment]
[ChartingState_OLD1.txt](https://github.com/KadeDev/Kade-Engine/files/6894577/ChartingState_OLD1.txt)

That is the charter from KE1.5.2, which I've slightly updated to work with KE1.6. Download the file and rename it ChartingState.hx and replace the one from 1.6. Remember to backup the 1.6 first just in case it's silly.

**Comment by isakube** [comment]
> [ChartingState_OLD1.txt](https://github.com/KadeDev/Kade-Engine/files/6894577/ChartingState_OLD1.txt)
> 
> That is the charter from KE1.5.2, which I've slightly updated to work with KE1.6. Download the file and rename it ChartingState.hx and replace the one from 1.6. Remember to backup the 1.6 first just in case it's silly.

No Event Editor lol
